frontend/src/pages/grants/index.tsx

```diff
@@ -87,13 +92,31 @@ export const GrantsPage = () => {
           my: 5,
         }}
       >
-        {status === DeadlineStatus.HappeningNow && (
-          <GrantForm conference={code} />
-        )}
-        {status === DeadlineStatus.InTheFuture && (
-          <GrantsComingSoon start={start} />
+        {isLoggedIn ? (
```

Comment:
what happens if we remove this condition?

Reply:
the page already redirect! I forgot to remove this!